internal/gcs/main.go

```diff
@@ -0,0 +1,76 @@
+// Package gcs provides a gRPC server implementation for the Blob Storage Interface (BSI).
+// Google Cloud Storage (GCS) is the implemented storage service.
+package gcs
+
+import (
+	"context"
+	"log"
+	"net"
+	"time"
+
+	bsi "github.com/substratusai/substratus/internal/bsi"
+
+	storage "cloud.google.com/go/storage"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// server implements the bsi.ControllerServer interface.
+type server struct {
+	bsi.UnimplementedControllerServer
```

Comment:
This line should probably go away right?

Reply:
My understanding is this helps with compatibility. If you add a new method to the UnimplementedControllerServer interface, your existing server will still run without necessarily needing to add those methods. Comments in the generated code are terse but allude to this:

```
// ControllerServer is the server API for Controller service.
// All implementations must embed UnimplementedControllerServer
// for forward compatibility
```
and
```
// UnimplementedControllerServer must be embedded to have forward compatible implementations.
```